src/i3_json_parser/i3_json_parser-utility.cpp

```diff
@@ -181,6 +181,22 @@ namespace
 
         return window;
     }
+
+    /**
+     * \brief                      Extract the fullscreen_mode of the window from the JSON object.
+     * 
+     * \param [in] a_json_object   JSON object.
+     * 
+     * \return                     Extracted fullscreen_mode.
+     */
```

Comment:
Please replace `Extract` with `Extracts` and remove trailing whitespaces in this paragraph.